### PR TITLE
Set contentful feature flags to true if PR is labelled `contentful`

### PR DESCRIPTION
**Pull request by lennym**

### CODE REVIEW

File: .github/actions/setup-environment/action.yml [hunk 1]
```diff
@@ -241,6 +241,7 @@ runs:
 
         if [ "$ENVIRONMENT_NAME" = "Branch" ]; then
           PR_NUMBER=${{ github.event.number || github.event.inputs.pr-number || inputs.pr-number }}
+          IS_CONTENTFUL=${{ contains(github.event.pull_request.labels.*.name, 'contentful') }}
```
Comment (gabiayako) [inline .github/actions/setup-environment/action.yml:244]: maybe HAS_CONTENTFUL_LABEL? only IS_CONTENTFUL sounds a little bit weird

**Review by gabiayako — APPROVED** [review]

File: .github/actions/setup-environment/action.yml (hunk 1 above)
Comment (peterstarling) [reply .github/actions/setup-environment/action.yml:244]: agreed, we have already too many of 'is_contentful's in the codebase

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: .github/actions/setup-environment/action.yml (hunk 1 above)
Comment (lennym) [reply .github/actions/setup-environment/action.yml]: Done.

**Review by peterstarling — APPROVED** [review]

**Pull request merged**